client/src/App.js

```diff
@@ -27,6 +28,8 @@ const App = () => {
         setUserCred(user);
         console.log('Signed in.');
       } else {
+        //make sure firebase auth login persistence is set to session only
+        rememberMe(false)
```

Comment:
I added this to the hook to reset persistence so long as you are signed out.
This way the checkbox is always accurately reflecting which option is selected on page load and there are fewer bugs regarding persistence.